OCP 4.15 Release Notes for P/Z

alishaIBM opened the pull request:
Version(s): 4.15

Issue: 
- https://issues.redhat.com/browse/MULTIARCH-3999

Link to docs preview:
- P: https://70664--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-ibm-power
- Z: https://70664--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-ibm-z

Draft link: 
- https://docs.google.com/document/d/1PH_dTQ-_9NJlzgK_ycGehQSToWUDoXnqb1kOMeb214M/edit

QE review:
- For P : Manoj Kumar, Luvella , Julie
- for Z : Holger Wolf, Florian Leber
- [x] QE has approved this change.

Branch : enterprise-4.15
@SNiemann15

SNiemann15 commented on release_notes/ocp-4-15-release-notes.adoc:
```diff
@@ -278,6 +344,10 @@ For more information, see xref:../installing/installing_ibm_cloud_public/user-ma
 |====
 |Feature |{ibm-power-name} |{ibm-z-name} and {ibm-linuxone-name}
 
+|cert-manager Operator for Red Hat OpenShift
```
```suggestion
|{cert-manager-operator}
```

SNiemann15 commented on release_notes/ocp-4-15-release-notes.adoc:
```diff
@@ -75,16 +75,70 @@ For more information, see xref:../installing/installing_ibm_cloud_public/user-ma
 [discrete]
 === {ibm-power-title}, {ibm-z-title}, and {ibm-linuxone-title} support matrix
 
+Starting in {product-title} 4.15, Extended Update Support (EUS) is extended to the {ibm-power-title} and the {ibm-z-title} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
```
```suggestion
Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to the {ibm-power-title} and the {ibm-z-title} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
```

SNiemann15 reviewed (changes requested):
@alishaIBM Only use the `{ibm-xxx-title}` attributes in titles or when referencing to titles. Otherwise use: `{ibm-xxx-name} `
I've left suggestions for IBM Z apply that to the Power part too.

prb112 commented on release_notes/ocp-4-15-release-notes.adoc:
```diff
@@ -80,16 +80,68 @@ For more information, see xref:../installing/installing_ibm_cloud_public/user-ma
+* {cert-manager-operator}
+* {ibm-power-name} Virtual Server Block CSI Driver Operator
+* Installer-provisioned Infrastructure Enablement for {ibm-power-name} Virtual Server
+* Multi-architecture Control Plane with support of Intel and Power workers
```
```suggestion
* Multi-architecture Power Control Plane with support of Intel and Power workers
```

alishaIBM replied:
Updated the PR. Thanks @prb112

SNiemann15 commented on release_notes/ocp-4-15-release-notes.adoc:
```diff
@@ -80,16 +80,68 @@ For more information, see xref:../installing/installing_ibm_cloud_public/user-ma
+
+For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_z/installing-ibm-z-kvm.html#installing-rhcos-using-ibm-secure-execution_installing-ibm-z-kvm[Installing a cluster with an LPAR on {ibm-z-name} and {ibm-linuxone-name}]
```
```suggestion
* xref:../installing/installing_ibm_z/installing-ibm-z-lpar.adoc#installing-ibm-z-lpar[Installing a cluster in an LPAR on {ibm-z-name} and {ibm-linuxone-name}]
* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z-lpar.adoc#installing-restricted-networks-ibm-z-lpar[Installing a cluster in an LPAR on {ibm-z-name} and {ibm-linuxone-name} in a restricted network]

```

SNiemann15 commented on release_notes/ocp-4-15-release-notes.adoc:
```diff
@@ -80,16 +80,68 @@ For more information, see xref:../installing/installing_ibm_cloud_public/user-ma
+For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_z/installing-ibm-z-kvm.html#installing-rhcos-using-ibm-secure-execution_installing-ibm-z-kvm[Installing a cluster with an LPAR on {ibm-z-name} and {ibm-linuxone-name}]
+//update above with the correct link when the PR is merged.
```
```suggestion

```

SNiemann15 commented on release_notes/ocp-4-15-release-notes.adoc:
```diff
@@ -80,16 +80,68 @@ For more information, see xref:../installing/installing_ibm_cloud_public/user-ma
+This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:
+
+* Agent-based Installer
+* {cert-manager-operator}
```
```suggestion
* {cert-manager-operator}
* `s390x` control plane with `x86_64` multi-architecture compute nodes
```

holgwolf commented:
LGTM

alishaIBM commented on release_notes/ocp-4-15-release-notes.adoc:
```diff
@@ -129,16 +129,70 @@ You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local
+* {cert-manager-operator}
+* {ibm-power-name} Virtual Server Block CSI Driver Operator
+* Installer-provisioned Infrastructure Enablement for {ibm-power-name} Virtual Server
+* Multi-architecture Power Control Plane with support of Intel and {ibm-power-name} workers
+* nx-gzip for Power10 (Hardware Acceleration)
+* openshift-install to support various SMT levels on {ibm-power-name} (Hardware Acceleration)
```
@prb112 request you to review above component names once again. The update is I have added {ibm-power-name}.

prb112 replied:
Looks good.

prb112 replied:
/lgtm

alishaIBM commented:
/assign @manojnkumar

alishaIBM commented:
/label peer-review-needed

fyi: QE approval pending.

xenolinux commented on release_notes/ocp-4-15-release-notes.adoc:
```diff
@@ -129,16 +129,70 @@ You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}
```
```suggestion
Compute nodes must run {op-system-first}.
```

xenolinux commented on release_notes/ocp-4-15-release-notes.adoc:
```diff
@@ -129,16 +129,70 @@ You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}
```
```suggestion
Compute nodes must run {op-system-first}.
```

xenolinux commented on release_notes/ocp-4-15-release-notes.adoc:
```diff
@@ -129,16 +129,70 @@ You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local
 
 [id="ocp-4-15-ibm-z"]
 === {ibm-z-title} and {ibm-linuxone-title}
+With this release, {ibm-z-name} and {ibm-linuxone-name} are now compatible with {product-title} {product-version}. The installation can be performed with z/VM, LPAR, or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see the following documentation:
```
```suggestion
With this release, {ibm-z-name} and {ibm-linuxone-name} are now compatible with {product-title} {product-version}. You can perform the installation with z/VM, LPAR, or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see the following documentation:
```

xenolinux commented on release_notes/ocp-4-15-release-notes.adoc:
```diff
@@ -129,16 +129,70 @@ You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local
+[discrete]
+==== Installing a cluster with an LPAR on {ibm-z-title} and {ibm-linuxone-title}
+
+{product-title} now supports user-provisioned installation of {product-title} 4.15 with a logical partition (LPAR) on {ibm-z-title} and {ibm-linuxone-title}
```
```suggestion
{product-title} now supports user-provisioned installation of {product-title} 4.15 with a logical partition (LPAR) on {ibm-z-title} and {ibm-linuxone-title}.
```

manojnkumar commented:
LGTM

alishaIBM commented:
/label merge-review-needed

fyi: QE has approved this change.

adellape commented on release_notes/ocp-4-15-release-notes.adoc:
```diff
@@ -129,16 +129,70 @@ You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local
+* Installer-provisioned Infrastructure Enablement for {ibm-power-name} Virtual Server
+* Multi-architecture Power Control Plane with support of Intel and {ibm-power-name} workers
+* nx-gzip for Power10 (Hardware Acceleration)
+* openshift-install to support various SMT levels on {ibm-power-name} (Hardware Acceleration)
```
Markup `openshift-install` and add something like utility/command/binary/tool after:
```suggestion
* The `openshift-install` utility to support various SMT levels on {ibm-power-name} (Hardware Acceleration)
```
I couldn't find a definitive answer for what to call it, but those were some options from around the repo.

adellape reviewed (commented):
Just one comment I'd like to hold merge on. Please re-queue for merge after addressing. Thank you!

alishaIBM commented:
/label merge-review-needed

> Just one comment I'd like to hold merge on. Please re-queue for merge after addressing. Thank you!

Addressed the comments. Re-queuing for merge. Thanks. FYI: @adellape

SNiemann15 commented on release_notes/ocp-4-15-release-notes.adoc:
```diff
@@ -266,15 +328,19 @@ For more information, see link:https://docs.openshift.com/builds/1.0/about/overv
 |Supported
 
-|{ibm-power-server-name} Block CSI Driver Operator (Technology Preview)
+|{ibm-power-server-name} Block CSI Driver Operator
```
@JoeAldinger If you look at the table headers you can see that column 2 is IBM Power, column 3 is IBM Z. 
The two features are supported on IBM Power but not on IBM Z and the plan is to move from TP to GA with 4.15.

JoeAldinger replied:
Sorry, my confusion.
